### DISCUSSION TITLE
【自定义排序功能】演示代码有问题,应该是  const [res, setRes] =  React.useState({ meta: [], data: [] });

### DISCUSSION THREAD

**Discussion by akunlee** [antvis/S2#2759] [General]
![image](https://github.com/antvis/S2/assets/52097543/541c4002-ec30-480d-bb97-97d40c5ea910)

**Comment by lijinke666** [comment]
感谢反馈, 近期修复

**Comment by lijinke666** [comment]
https://github.com/antvis/S2/pull/2760
